artman/config/proto/config_pb2.py

```diff
@@ -0,0 +1,569 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
That was my original instinct. I failed to find a doc or example on how to do that. Can you give me some guidance? The official python protobuf doc (https://developers.google.com/protocol-buffers/docs/pythontutorial) has very limited information regarding the distribution (proto vs pb2).